docs/channels.md

```diff
@@ -132,6 +132,18 @@ session = rocket_server.login('username', 'password')
 channels = session.channels.list(query: {usernames: 'friend-username'})
 ```
 
+#### channels.list.joined
+
+Lists all of the channels the calling user has joined. 
+list is also used for searching/querying_
+
+```ruby
+require 'rocketchat'
+
+rocket_server = RocketChat::Server.new('http://your.server.address/')
+session = rocket_server.login('username', 'password')
+channels = session.channels.list_joined(query: {usernames: 'friend-username'})
```

Comment:
Please take a look at https://github.com/bbatsov/ruby-style-guide#spaces-braces

Specifically putting spaces inside hashes

Reply:
I have suggest answer for your opinion.
```
channels = session.channels.list_joined(query: { usernames: 'friend-username' })
```
what do you think?
actually, i have little bit confusing.
I mean, that's document code to same with [channels list api doc](https://github.com/abrom/rocketchat-ruby/blob/master/docs/channels.md#channelslist)

Reply:
That looks right. Regarding the channels list api doc, that one needs to be fixed ;)

Reply:
yeah~ Thanks, i will fix it, soon :)